src/Ingredient.java

```diff
@@ -0,0 +1,40 @@
+public class Ingredient {
+    private double amount;
+    private String unit;
+    private String name;
+
+//    Ik print nu voor ieder ingredient een double uit omdat ik een double nodig heb voor de hoeveelheid van de appels.
+//    Maar dit wil ik natuurlijk niet voor alle ingredienten doen. Ik weet alleen even niet hoe ik de rest van de
+//    ingredienten als een int moet instantieren en de appels als een double.
```

Comment:
Ik heb dit besproken in het huiswerkklas deel 2.2 (maar je zou modulo kunnen gebruiken)